packages/liquidsoap/liquidsoap.1.4.2/opam

```diff
@@ -124,9 +124,10 @@ conflicts: [
   "portaudio" {< "0.2.0"}
   "pulseaudio" {< "0.1.2"}
   "samplerate" {< "0.1.1"}
+  "sdl-liquidsoap" {>= "2"}
```

Comment:
So in these you really want the new package with less dependencies?

Reply:
I think so. We switched to `tsdl` for the upcoming release and either some functionalities went away or the packages are structured differently but we don't seem to need the `mixer` part now.

Reply:
> The opposite! I'm excluding the upcoming releases (the field is for conflicts with optional dependencies).

Oh sorry. Indeed. I was biased because I thought to remember that conflicts behave weird with and and or clauses. @kit-ty-kate are the conflicts fine as is or they need to be split in two separate lines? Like

    "ffmpeg" {< "0.2.0"}
    "ffmpeg" {>= "0.5.0"}

Reply:
Looking at other examples it does seem like you are right. Just pushed an updated PR.